modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -7,19 +7,49 @@ Before deploying a Mule application to Anypoint Runtime Fabric, determine the co
+
+Guaranteed CPU allocation allows applications to perform consistently by ensuring there is no CPU contention between other applications. It gives guaranteed performance in all cases.
+
+When the `vCPU Limit` is set to a higher value than the `Reserved vCPU`, the application can burst up to the either the vCPU limit value or the amount of unreserved vCPU on the worker node, which ever is lesser.
+
+When using CPU bursting, you must consider the following factors:
+
+* CPU Bursting Limitations
+
+** Minimum reserve: 0.02 vCPU.
+** Minimum limit: 0.07 vCPU. This aligns with the minimum for Runtime Fabric v1.3 and earlier.
+** The CPU limit is bounded by the CPU cores provided on the worker nodes.
+** The maximum suggested utilization per CPU core is 20 - 25 simple apps and API gateways. More complex apps that are frequently used should get more capacity. When the minimum reserve and minimum limit are specified, this provides approximately .5 cores to share across all apps when needed.
+* Runtime Fabric runs a small amount of services on worker nodes. These services consume between 0.3 and 0.5 cores, depending on whether log forwarding is enabled for Anypoint Monitoring.
+* CPU contention. With CPU bursting, applications can compete for a shared resource- the unallocated CPU remaining on the worker node. As more apps are deployed on a worker node, each app has less unallocated CPU to share with other nodes, apps can suffer from poor performance due to not getting enough CPU. The following strategies can be considered for minimizing CPU contention with your applications:
+
+** Deploy intensive nightly batch applications with API that have peak load during the day.
+** Deploy multiple replicas of an application so that they are distributed across worker nodes to maximize the surface area for the unallocated CPU.
++
+Regardless of what approach is chosen, make sure you run tests to validate performance.
+* The number of replicas and the CPU and memory configuration do not change dynamically, so there is no horizontal or vertical autoscaling. 
+* To achieve high availability with deployed applications, deploy multiple replicas. 
+* When sizing your worker nodes, make sure to keep your CPU usage in compliance with your Mule license. Mule licensing is based upon the amount of CPU cores Mule apps and API gateways have access to use. 
+
 
 == Minimum Core and Memory Requirements
 
 The minimum values for the amount of CPU and memory allocated to each replica of a Mule application are:
 
 * vCPU Cores: 0.07 cores
-* Memory: 0.7 GiB memory (Mule 4) or 0.5 GiB (Mule 3.x)
+* Memory: 0.7 GiB memory (Mule 4) or 0.5 GiB (Mule 3)
 
 == Calculating Memory Allocation on CloudHub and Runtime Fabric
```

Comment:
ORG:
Be sure that this topic is introduced in the text of its container topic.
Be sure that this topic is basically parallel with and a peer of other topics at this level.
Be sure that this topic text introduces any subtopics that it contains.